grafana-plugin/src/GrafanaPluginRootPage.tsx

```diff
@@ -150,12 +151,20 @@ export const Root = observer((props: AppRootProps) => {
     onNavChanged(navModel);
   }, [navModel, onNavChanged]);
 
-  const Page = pages.find(({ id }) => id === page)?.component || pages[0].component;
+  const { action: pagePermissionAction, component: PageComponent } = pages.find(({ id }) => id === page) || pages[0];
+  const userHasAccess = pagePermissionAction ? !!grafanaUser.permissions[pagePermissionAction] : true;
 
   return (
     <DefaultPageLayout {...props}>
       <GrafanaTeamSelect currentPage={page} />
-      <Page {...props} path={pathWithoutLeadingSlash} />
+      {userHasAccess ? (
+        <PageComponent {...props} path={pathWithoutLeadingSlash} />
+      ) : (
+        <>
+          {/* TODO: what should be the behavior here? a redirect? if so, to what page? or maybe an unauthorized component? */}
+          <p>You don't have access!</p>
```

Comment:
agree, some additional info, links can be useful here, I suppose @raphael-batte can help

Reply:
@raphael-batte the current UI would look like this. Would you prefer that users get redirect to some page, or that they see some unauthorized message/component?
<img width="1453" alt="Screen Shot 2022-10-24 at 15 05 34" src="https://user-images.githubusercontent.com/9406895/197532593-2574820c-5de5-46e2-8d43-2a80023f2f63.png">

Reply:
looks nice!
one little thing - lets correct color for 403
basic Grafana orange will be great here - rgb(255, 120, 10)

Reply:
@raphael-batte is it okay if I create a separate issue to update this?

Currently this is using the same color as a bunch of other places in the UI (#f8d06b for dark theme, #8a6c00 for light).

Reply:
@raphael-batte I created #768 to make sure we don't forget about this